### ISSUE TITLE
[ENHANCEMENT] Limit notification length (twitter)

### ISSUE THREAD

**Issue by dsorc** [kvangent/PokeAlarm#359]
## Enhancement Description
Notifications that are too long cannot be sent on twitter, some spawns are missed this way.
Main culprit are the google maps reverse text substitutions (other tags have a known max length). I have seen some street names with 45 characters.

What would be needed:
- a max notification length variable for twitter (the rest would be truncated)
- or a way to limit individual tags to a specific number of characters, like <street.15> or <street%15> (probably doesn't work in json)

## Use cases
You can include more information in your tweet without the risk of missing some spawns.
My current notification text is the following, usually about 100 char long, and I'm not even sure it is short enough.
```
"pokemon": {"status": "<iv>% <pkmn> on <street>, <atk>/<def>/<sta> IV, <quick_move>/<charge_move>! Exp <24h_time> <gmaps>"}
```

**Comment by renxida** [comment]
This would be a very cool thing to do.

**Comment by dsorc** [comment]
Until this gets implemented I manually shortened the move names in
\locales\en\moves.json

**Comment by kvangent** [comment]
Implemented twitter truncation in dfc9e9c348a0d238716dd464ac3177fbbe9ad4bb.
